app/docs/0.12.x/admin-api.md

```diff
@@ -1787,6 +2025,111 @@ Attributes | Description
 HTTP 204 No Content
 ```
 
+---
+
+### Set target as healthy
+
+Set the current health status of a target in the load balancer to "healthy".
+This resets the health counters of the health checkers running in all workers
+of the kong node, and broadcasts a cluster-wide message so that this
+health status is propagated to the whole Kong cluster.
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> This endpoint is only available with Kong 0.12+
+</div>
+
+#### Endpoint
+
+<div class="endpoint delete">/upstreams/{upstream name or id}/targets/{target or id}/healthy</div>
+
+Attributes | Description
+---:| ---
+`upstream name or id`<br>**required** | The unique identifier **or** the name of the upstream for which to delete the target.
+`target or id`<br>**required** | The host/port combination element of the target to remove, or the `id` of an existing target entry.
+
+#### Response
+
+```
+HTTP 204 No Content
+```
+
+---
+
+### Set target as healthy
+
+Set the current health status of a target in the load balancer to "healthy"
+in the entire Kong cluster.
+
+This endpoint can be used to manually re-enable a target that was previously
+disabled by the upstream's [health checker](healthchecks). Upstreams only
+forward requests to healthy nodes, so this call tells Kong to start using this
+target again.
+
+This resets the health counters of the health checkers running in all workers
+of the Kong node, and broadcasts a cluster-wide message so that the "healthy"
+status is propagated to the whole Kong cluster.
+
+<div class="alert alert-warning">
```

Comment:
Is this note necessary? These docs will appear in v 0.12 docs section - do we typically mention in docs that new features are new as of the current version?

Reply:
Ah, I saw this alert in another endpoint, but now that I double check it was for an endpoint introduced in a minor release(!). Will remove this.